enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -214,4 +259,24 @@ public RoleResult( String role, Set<String> users )
             this.users.addAll( users );
         }
     }
+
+    public class TransactionResult
+    {
+        public final String username;
+        public final String transaction;
+
+        public TransactionResult( KernelTransaction tx )
+        {
+            if ( tx.mode() instanceof AuthSubject )
+            {
+                AuthSubject authSubject = (AuthSubject) tx.mode();
+                username = authSubject.name();
+            }
+            else
+            {
+                username = "-";
+            }
+            transaction = tx.toString();
```

Comment:
Ok, I guess we go by the transaction count until someone request something else. Will fix this next week.